packages/slice-machine/src/components/VideoPopover/VideoPopover.stories.tsx

```diff
@@ -39,3 +39,23 @@ export const VideoPlayer = {
     />
   ),
 };
+
+export const CloseWhenClickingOutSide = {
+  render: () => {
+    return (
+      <div>
+        <VideoPopover
+          open={true}
+          onClose={noop}
+          onPlay={noop}
+          cloudName="dmtf1daqp"
+          publicId="Tooltips/pa-course-overview_eaopsn"
+          delay={5000}
```

Comment:
For the delay in storybook, for all stories I would put something really fast like 1 or 2 seconds, so we don't wait as long haha, WDYT?

Reply:
1.5